src/fides/api/ops/models/system_manager.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import Column, ForeignKey, String
+
+from fides.lib.db.base_class import Base
+
+
+class SystemManager(Base):
+    """Many to many table to link users to systems as system managers.
+
+    Usually access via user.systems or system.users instead of via this table directly.
+    """
+
+    user_id = Column(String, ForeignKey("fidesuser.id"), primary_key=True)
+    system_id = Column(String, ForeignKey("ctl_systems.id"), primary_key=True)
```

Comment:
Instead of accessing this many-to-many table directly, you'd likely use the FidesUser methods instead.